docs/metrics/getting-started-prometheus-grafana/README.md

```diff
@@ -91,54 +120,20 @@ while (!Console.KeyAvailable)
 }
 ```
 
-After the above modifications, now our `Program.cs` should look like [this](./Program.cs).
-
-### Check results in the browser
-
-Start the application and keep it running. Now we should be able to see the
-metrics at [http://localhost:9464/metrics](http://localhost:9464/metrics) from a
-web browser:
-
-![Browser UI](https://user-images.githubusercontent.com/17327289/151633547-736c6d91-62d2-4e66-a53f-2e16c44bfabc.png)
-
-Now, we understand how we can configure `PrometheusHttpListener` to export metrics.
-Next, we are going to learn about how to use Prometheus to collect the metrics.
-
 ## Collect metrics using Prometheus
 
+### Install and run Prometheus
+
 Follow the [first steps](https://prometheus.io/docs/introduction/first_steps/)
 to download the [latest release](https://prometheus.io/download/) of Prometheus.
 
-### Configuration
-
 After finished downloading, extract it to a local location that's easy to
-access. We will find the default Prometheus configuration YAML file in the
-folder, named `prometheus.yml`.
-
-Let's create a new file in the same location as where `prometheus.yml` locates,
-and named the new file as `otel.yml` for this exercise. Then, copy and paste the
-entire content below into the `otel.yml` file we have created just now.
-
-```yaml
-global:
-  scrape_interval: 10s
-  evaluation_interval: 10s
-scrape_configs:
-  - job_name: "otel"
-    static_configs:
-      - targets: ["localhost:9464"]
-```
+access. Run the `prometheus(.exe)` server executable with feature flag
+[otlp-receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
+enabled:
 
-### Start Prometheus
-
-Follow the instructions from
-[starting-prometheus](https://prometheus.io/docs/introduction/first_steps/#starting-prometheus)
-to start the Prometheus server and verify it has been started successfully.
-
-Please note that we will need pass in `otel.yml` file as the argument:
-
-```console
-./prometheus --config.file=otel.yml
+```sh
+./prometheus --enable-feature=otlp-write-receiver
 ```
 
 ### View results in Prometheus
```

Comment:
@reyang Something I noticed (not related to a change in this PR). This instruction says 
> type `MyFruitCounter` in the expression bar of the UI; finally, click the execute

But when i run it, it is `MyFruitCounter_total` . see [screenshot](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4874#issuecomment-1731748476)

Reply:
You're right, I've updated the docs (I didn't change the screenshot though).